### ISSUE TITLE
Useless % and ^ operators

### ISSUE THREAD

**Issue by sobolevn** [wemake-services/wemake-python-styleguide#1573]
# Bug report

## What's wrong

This code does not raise any violations:`

```python
3 ^ 1
3 ^ 0
3 % 1
3 % 0
```

## How is that should be

It should raise:
- useless operations for `3 ^ 1`, `3 ^ 0` and `3 % 1`
- zero division for `3 % 0`

**Comment by SHITIZ-AGGARWAL** [comment]
hello @sobolevn 
I am new to all this, and would very much like to work on this issue
can you please guide me and tell how to get started

**Comment by sobolevn** [comment]
I would love to! @SHITIZ-AGGARWAL 

Start with reading our contribution guide: https://github.com/wemake-services/wemake-python-styleguide/blob/master/CONTRIBUTING.md

We need to add these new cases here: https://github.com/wemake-services/wemake-python-styleguide/blob/master/wemake_python_styleguide/visitors/ast/operators.py#L40

And test them here: https://github.com/wemake-services/wemake-python-styleguide/blob/master/tests/test_visitors/test_ast/test_operators/test_useless_math.py

**Comment by hhsdev** [comment]
Hey, can I take this one? Thanks.

**Comment by sobolevn** [comment]
Yes, sure!
